test/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+asyncio_mode = auto
+testpaths = integration
+filterwarnings =
+    ignore::schematics.deprecated.SchematicsDeprecationWarning
```

Comment:
Why do you ignore deprecation message?

Reply:
There are additional warnings generated by Schematics because some features are deprecated but are still being used